content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -291,6 +308,32 @@ Here, we first check that the optional parameter is present with `!has(params.op
   evaluated, and optionalNumber will be checked to ensure that it contains a value between 5 and
   10 inclusive.
 
+
+##### Per-Namespaced Parameters
+
+Authors of policy bindings may choose to specify cluster-wide, or per-namespace
+parameters. Normally, parameters will be searched for only in the `namespace` provided
+in the binding's `paramRef`.
+
+However, if `namesapce` is unset, the `namespace` for the request object can be
```

Comment:
Consider that "unset" itself can be used as a verb, I'd suggest we say "is not set".

Reply:
changed all instances of unset to "not specified"